### PR TITLE
Issue 2047

### PR DESCRIPTION
**Pull request by hootiehoo**
TAs can now see the total number of assignments they have been assigned, as well as number of assignments they have finished marking.
Admin can see this information for all TAs on the dashboard in the summaries box.

### CODE REVIEW

File: app/views/submissions/_submissions_table.js.jsx.erb [hunk 1]
```diff
@@ -157,6 +157,11 @@
     render: function() {
       // Do student-specific table stuff here.
       // possibly move this to getdefaultprops.
+      num_completed_assignments = this.state.submissions.filter(function(submission) {
+                return (submission['marking_state'].indexOf("Completed") > -1);
```
Comment (arkon) [inline app/views/submissions/_submissions_table.js.jsx.erb]: This won't work with other languages.

Comment (hootiehoo) [inline app/views/submissions/_submissions_table.js.jsx.erb]: You're right. I hadn't thought of that. I will correct it.

Comment (david-yz-liu) [inline app/views/submissions/_submissions_table.js.jsx.erb]: @hootiehoo You should do the same thing you're doing for the Admin dashboard page.

File: app/views/assignments/_grader_summary.html.erb [hunk 2]
```diff
@@ -6,7 +6,7 @@
 <% else %>
   <ul>
     <% assignment.tas.each do |ta| %>
-      <li><%= "#{ta.first_name} #{ta.last_name}" %></li>
+      <li><%= "#{ta.first_name} #{ta.last_name}" %> (<%= "#{ta.get_num_marked(assignment)}\/#{ta.get_num_assigned(assignment)}" %> marked). </li>
```
Comment (david-yz-liu) [inline app/views/assignments/_grader_summary.html.erb]: Break up this line.

File: app/views/submissions/browse.html.erb [hunk 3]
```diff
@@ -78,6 +78,11 @@
                       groupings: @groupings),
                     confirm: t('collect_submissions.marking_incomplete_warning') %>
     <% end %>
+    <% if @current_user.ta? %> |
+      <%= t('browse_submissions.how_many_marked',
+              num_marked: @current_user.get_num_marked(@assignment),
```
Comment (david-yz-liu) [inline app/views/submissions/browse.html.erb]: Align the arguments, so `num_marked` is aligned with `'browse`

**Comment by david-yz-liu** [comment]
Thanks, @hootiehoo

**Pull request merged**